roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -200,7 +206,7 @@ def __init__(self, subject_string):
         self.subjects = []
         for s in subject_string.split(', '):
             name, _, value = s.partition('=')
-            self.subjects.append((name, value))
+            self.subjects.append((name.encode(), value.encode()))
```

Comment:
Same story with implicit x explicit encoding.

A general advice is to decode strings when reading, using unicode strings (Py2) and strings (Py3) everywhere internally, and encode only once writing.

https://docs.python.org/2/howto/unicode.html#tips-for-writing-unicode-aware-programs
https://docs.python.org/3/howto/unicode.html#tips-for-writing-unicode-aware-programs